examples/Polynomial_Optimization.ipynb

```diff
@@ -184,31 +242,37 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 6,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "JuMP.terminationstatus(m) = Success::MathOptInterface.TerminationStatusCode = 0\n",
-      "JuMP.objectivevalue(m) = -2.0092666419557759e-10\n"
+      "termination_status(m) = OPTIMAL::TerminationStatusCode = 1\n",
+      "objective_value(m) = -2.0092710828478744e-10\n"
      ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "-2.0092710828478744e-10"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
     }
    ],
    "source": [
-    "using JuMP\n",
-    "using SumOfSquares\n",
-    "const MOI = MathOptInterface\n",
-    "MOI.empty!(optimizer)\n",
-    "m = SOSModel(optimizer = optimizer)\n",
-    "@variable m α\n",
-    "@objective m Max α\n",
-    "c3 = @constraint m p >= α domain = S\n",
+    "m = SOSModel(optimizer)\n",
```

Comment:
`m` -> `model`

Reply:
This should be in the style guide: http://www.juliaopt.org/JuMP.jl/v0.19.0/style/

Reply:
This isn't mentioned in the style guide. Is there a specific reason for following this convention?

Reply:
See https://github.com/JuliaOpt/JuMP.jl/issues/1885